frontend/pages/ManageControlsPage/OSSettings/cards/CustomSettings/components/ProfileListItem/ProfileListItem.tsx

```diff
@@ -111,14 +132,9 @@ const ProfileListItem = ({
         </div>
       </div>
       <div className={`${subClass}__actions-wrap`}>
-        {isPremium && !!labels?.length && (
-          <div className={`${subClass}__labels`}>
-            {labels?.some((l) => l.broken) && <Icon name="warning" />}
-            <LabelCount className={subClass} count={labels.length} />
-          </div>
-        )}
+        {renderLabelInfo()}
         <div className={`${subClass}__actions`}>
-          {isPremium && !!labels?.length && (
+          {isPremium && labels !== undefined && labels.length && (
```

Comment:
Would `labels?.length` work for both checking for `undefined` and `length`?

Reply:
it would change the check to `!!labels?.length`, which I think is a harder to read IMO.